### ISSUE TITLE
Scrambled textures after loading a resourcepack

### ISSUE THREAD

**Issue by kingarthur0975** [octarine-noise/BetterFoliage#372]
Any time I try to switch resourcepacks with this mod installed, the mod scrambles every texture that it has access to (grass blocks, tall and double tallgrass, leaves, etc.)
I've tried removing every other mod in my mods folder, and still no dice. I didn't see a fix for this in the changelog for 2.6.5.
For clarification, this issue is on version 2.6.4.

**Comment by lynbutnot** [comment]
I've seen people talking about the same issue—from what I've gathered it happens whenever the game has to reload (a.k.a. when the loading screen shows up). Have you tried exiting the game and going back in after you've switched the resource pack? That should make the textures show up correctly.

**Comment by kingarthur0975** [comment]
Yes, restarting the game does work. I just dont think you should have to restart the whole friggin' game any time you wanna try out a shader pack, or a texturepack. I honestly think it might be an issue with the porting of the mod, since it was originally fo older versions of Minecraft, versions before the loading screen.

**Comment by OcrSu** [comment]
i also have this bug in my modpack。it is uncomfortable.

**Comment by MrRedstoneA1** [comment]
This is pretty much normal with mods. Some mods will also scramble their own textures after resource pack change, especially entities. You should expect to restart the game after loading another resource pack, I'm afraid there's no way around that. You could also merge resource packs you like into one pack.
